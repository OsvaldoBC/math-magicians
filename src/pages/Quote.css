.quote {
  font-size: 2em;
  text-align: center;
  margin-top: 4em;
  width: 60%;
  background-color: #fff;
  padding: 2em;
  color: #858694;
  border-radius: 3em;
  box-shadow: 5px 10px 10px 10px #e0e0e0;
}