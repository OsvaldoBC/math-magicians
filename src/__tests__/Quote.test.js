import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

test('match home snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
