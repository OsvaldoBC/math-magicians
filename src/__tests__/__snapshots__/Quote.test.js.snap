// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match home snapshot 1`] = `
<div>
  <p
    className="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: It is about understanding. - William Paul Thurston
  </p>
</div>
`;
